### ISSUE TITLE
Differentiate between validations and labeled objects

### ISSUE THREAD

**Issue by bbernhard** [bbernhard/imagemonkey-core#226]
Currently, there is no differentiation between validations and labeled objects. i.e if someone adds a label to an image it will show up as validation in the activity chart. we should change that. 

PoC:

![activity_1](https://user-images.githubusercontent.com/5710631/55423198-b7891c80-557d-11e9-8686-403354f6bbdd.png)

**Comment by dobkeratops** [comment]
I was going to suggest adding image uploads to the activity chart aswell, but I recall the earlier discussion on having so many different parts to that chart (possible workaround being a dropbox to select activity type, default = "total", then you could narrow down to see the individual constituents; the reason for wanting image uploads is they still show you someone is contributing to the site, even if they aren't devoting time directly on that day)

**Comment by bbernhard** [comment]
you are right, the image uploads should definitely be in the chart as well.

What we could also do (in case we want to stick with a single activity chart) is to use to stack the number of image uploads on top of another value (stacked bar chart). We already have a few activity types that will be stacked on top of each other (e.g: if you look at the "2019-03-17" in the activity chart you will see that the labeled objects are stacked on top of the image descriptions). We could do something similar for the image uploads.

While playing with the activity chart, I realized that the max. number of bars that we can place next to each other is 3. If there are more than 3 bars, then the activity chart will become to big. That's not so much of a problem on large screens, but on smaller devices (like smartphones) you can't see anything anymore. 



But of course, we could also go with a dropdown to filter
